wd_capabilities.au3

```diff
@@ -14,9 +14,11 @@
 ; Language ......: English
 ; Description ...: A collection of functions used to dynamically build the Capabilities string required to create a WebDriver session
 ; Author ........: mLipok
-; Modified ......:
-; URL ...........:
-; Date ..........: 2022/04/07
+; Modified ......: Danp2
+; URL ...........: https://github.com/Danp2/au3WebDriver/blob/master/wd_capabilities.au3
+; URL ...........: https://w3c.github.io/webdriver/#capabilities
+; URL ...........: https://docs.yugabyte.com/preview/api/ysql/datatypes/type_json/primitive-and-compound-data-types
```

Comment:
I know you are trying to be thorough, but do we really need all of these URLs? IMO, the one to the W3C specs should be enough.

Reply:
Ok

Reply:
Take a look here: [#INDEX# URL](https://github.com/Danp2/au3WebDriver/pull/285/commits/d6b9f2e4764bf495a35f6375030675cef3f1722f)

Reply:
I see that you've added another URL pointing to the wiki. Do we still need the first one since that page also includes a link to the W3C site?

Reply:
good point

Reply:
Take a look on:
[URL in UDF header](https://github.com/Danp2/au3WebDriver/pull/285/commits/c03201392e32abe0de21e2596ec6c2f304ad1be9)